nuxeo-full-ps-template.yaml

```diff
@@ -0,0 +1,1989 @@
+            value: ${APPLICATION_NAME}-elasticsearch:9200
+          - name: NUXEO_ES_CLUSTERNAME
+            value: ${ELASTICSEARCH_CLUSTER_NAME}
+          - name: NUXEO_ES_REPLICAS
+            value: "0"
+          - name: NUXEO_BINARY_STORE
+            value: /binaries
+
+          ports:
+          - containerPort: 8080
+            protocol: TCP
+          volumeMounts:
+          - name: ${APPLICATION_NAME}-binaries
+            mountPath: /binaries
+          - name: nuxeoconf
+            mountPath: /docker-entrypoint-initnuxeo.d
+          - name: connect-secret
+            mountPath: /opt/nuxeo/connect
+
+          - name: nuxeodata
+            mountPath: /var/lib/nuxeo/data
+          - name: nuxeolog
+            mountPath: /var/log/nuxeo
+          - name: nuxeotmp
+            mountPath: /opt/nuxeo/server/tmp
+        volumes:
+        - name: ${APPLICATION_NAME}-binaries
+          persistentVolumeClaim:
+            claimName: ${APPLICATION_NAME}-uat-binaries
+        - name: nuxeoconf
+          configMap:
+            name: ${APPLICATION_NAME}-uat-conf
+        - name: connect-secret
+          secret:
+            secretName: ${APPLICATION_NAME}-connect
+        - name: nuxeodata
+          emptyDir: {}
+        - name: nuxeolog
+          emptyDir: {}
+        - name: nuxeotmp
+          emptyDir: {}
+        dnsPolicy: ClusterFirst
+        restartPolicy: Always
+        terminationGracePeriodSeconds: 30
+    test: false
+    triggers:
+    - type: ConfigChange
+    - type: "ImageChange"
+      imageChangeParams:
+        automatic: false
+        containerNames:
+        - ${APPLICATION_NAME}-appserver
+        from:
+          kind: ImageStreamTag
+          name: ${APPLICATION_NAME}-nuxeo:latest
+
+- apiVersion: v1
+  kind: Service
+  metadata:
+    name: ${APPLICATION_NAME}-uat-nuxeo
+    labels:
+      app: ${APPLICATION_NAME}-uat
+      component: nuxeo
+      role: lb
+  spec:
+    clusterIP: None
+    selector:
+      app: ${APPLICATION_NAME}-uat
+      component: nuxeo
+      stage: uat
+    ports:
+    - name: 8080-tcp
+      port: 8080
+      protocol: TCP
+
+- apiVersion: v1
+  kind: Route
+  metadata:
+    labels:
+      app: ${APPLICATION_NAME}-uat
+    name: ${APPLICATION_NAME}-uat-nuxeo-route
+  spec:
+    # TODO: Add parameter
+    host: ${APPLICATION_NAME}-uat.${DOMAIN_SUFFIX}
+    tls:
+      termination: edge
+      insecureEdgeTerminationPolicy: Redirect
+    to:
+      kind: Service
+      name: ${APPLICATION_NAME}-uat-nuxeo
+    port:
+      targetPort: 8080-tcp
+    wildcardPolicy: None
+
+
+
+############################################################################################################
+# Build pipeline
+############################################################################################################
+
+
+- apiVersion: v1
+  kind: ImageStream
+  metadata:
+    name: ${APPLICATION_NAME}-base-image
+    labels:
+      app: ${APPLICATION_NAME}
+  spec: {}
+  status:
```

Comment:
Remove `status` as it's not part of the definition

Reply:
removed